atlp-rwanda/trojans-ec-fe
# #184760044 - Single and all product view

### What does this PR do?

Adds the viewing products pages both on the seller's dashboard and on the buyer's perpespective

### Description of Task to be completed?

A seller and buyer should be able to view all products and a single product

### How should this be manually tested?

Mainly on the deployed site and npm test
PS: You have to be logged in to see the products
One of the user one can use testBuyer@example.com with password of default


### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/n/projects/2628461/stories/184760044

### Screenshots (if appropriate)
### Figma
#### Main page all products
![figma-all-products](https://user-images.githubusercontent.com/111729060/231755139-8e6c0a13-a9b4-4781-a5d0-429f56e17610.PNG)
#### Main page single view
![figma-single-main](https://user-images.githubusercontent.com/111729060/231760027-7649c85d-5a2f-48ff-85c2-94d19674ff70.PNG)

#### Seller all products view
![figma-seller-all](https://user-images.githubusercontent.com/111729060/231755584-18039243-e1bf-43e2-adfe-c1711d427b93.PNG)
#### Seller single products view
![figma-seller-single](https://user-images.githubusercontent.com/111729060/231755900-7028455a-ee33-4a4c-b20f-87fcbdc43290.PNG)




### Actual
#### Main all products view
![allproducts](https://user-images.githubusercontent.com/111729060/231756137-f93b2c64-b348-4a78-98c7-3fa24bbbaa0b.PNG)
#### Main single product view
![singleMain](https://user-images.githubusercontent.com/111729060/231756366-6b3e7892-f528-463b-8b41-0e71dcbcbf2d.PNG)
#### Seller all products view
![sellerAll](https://user-images.githubusercontent.com/111729060/231756543-708bf0fd-16ce-4078-bf9f-adc5bbc48370.PNG)
#### Seller single products view
![sellerSingle](https://user-images.githubusercontent.com/111729060/231756842-7893c73a-1c40-4d50-aeb2-1c6d473695ff.PNG)



### Questions:

# Type of change

Please check the options that are relevant.


# Checklist:
